connectors/bitcoin.go

```diff
@@ -84,6 +87,22 @@ func (btc *BTC) Connect(endpoint string, username string, password string) error
 	if err != nil {
 		return fmt.Errorf("RPC client error: %v", err)
 	}
+
+	info, err := c.GetNetworkInfo()
```

Comment:
If I remember correctly, c.New fails if it cannot establish a connection. Why is this necessary?

Reply:
looks like it doesn't. I was able to start the server without spinning up a bitcoind node.

Reply:
Indeed the fact that one was able to run the app without having a btc node available is something I was aware of. The issue ends up blowing up when the user calls the acceptQuote

Reply:
Ok, I didn't know this